.github/workflows/template/build-{{ostype}}.yaml.j2

```diff
@@ -246,6 +246,27 @@ jobs:
             git-annex[-_]*.*
             dist/build-version
 
+      - name: Create new release
+        if: github.event.inputs.commitish != ''
+        run: |
+          if git rev-parse refs/tags/"$INPUT_COMMITISH" &> /dev/null
+          then
+              echo "[INFO] Building a tag; uploading assets to release ..."
+              if gh release list | grep -Fqx "$INPUT_COMMITISH"
+              then
+                  echo "[INFO] Release already exists; uploading assets"
+                  gh release upload "$INPUT_COMMITISH" git-annex[-_]*.*
+              else
+                  echo "[INFO] Creating release"
+                  gh release create "$INPUT_COMMITISH" git-annex[-_]*.*
+              fi
```

Comment:
placing possible race condition aside (let's consider unlikely to happen) -- shouldn't `gh release upload` happen also after `gh release create`?  i.e. if eg. linux first finished building, it should first create release and then still upload the build.

Reply:
[The `create` command](https://cli.github.com/manual/gh_release_create) supports uploading assets as part of the release creation.